policy/approval/approve.go

```diff
@@ -102,6 +103,12 @@ func (r *Rule) Evaluate(ctx context.Context, prctx pull.Context) (res common.Res
 		res.Status = common.StatusApproved
 	} else {
 		res.Status = common.StatusPending
+
+		if r.Options.RequestReview {
+			// we don't use organizations, because that would be far too noisy
+			res.RequestedUsers = r.Requires.Users
+			res.RequestedTeams = r.Requires.Teams
```

Comment:
I think we may also want options like `RequestAdminReview` and `RequestWriteCollaboratorReview`, since these are handled slightly differently